s.py

```diff
@@ -0,0 +1,229 @@
+import csv
+from dataclasses import dataclass
+from typing import Optional, List, Union, Dict
+
+import requests
+from bs4 import BeautifulSoup
+
+
+@dataclass
+class Everyone:
+    id: int
+    profile_url: str
+    username: Optional[str]
+    name: Optional[str]
+    github_username: Optional[str]
+    twitter_username: Optional[str]
+    location: Optional[str]
+    joined_at: Optional[Union[str, int]]
+    linked_in: Optional[str]
+    email: Optional[str]
+    work: Optional[str]
+    stackOverflow: Optional[str]
+    instagram: Optional[str]
+    facebook: Optional[str]
+    youtube: Optional[str]
+    education: Optional[str]
+    github_url: Optional[str]
+    twitter_url: Optional[str]
+    work_status: Optional[str]
+
+
+def get_profile_url() -> List[Everyone]:
+
+    code = requests.get(f"https://dev.to/api/users/1")
```

Comment:
We should make sure we're getting a positive response from the API

```suggestion
    devto_profile_response = requests.get(f"https://dev.to/api/users/1")
    
    if devto_profile_response.status_code != 200:
       log.info(f"Got a {devto_profile_response.status_code} response from Dev.to.\n
       Reason: devto_profile_response.reason")
       abort()
```